improve contributor help documentation

jansengb opened the pull request.

GeraldJansen committed:
improve contributor help documentation (#7)

GeraldJansen committed:
fix line breaks in contrib_help

John-Colvin commented on content/main/contrib_help.md:
```diff
@@ -0,0 +1,28 @@
+
+## Contributing to DlangScience
+
+All the packages which make up DlangScience are hosted in their own github projects at https://github.com/DlangScience/package. For the time being, development is proceeding quite independently in the projects. For example, issues are tracked separately within the respective Github site and each project can be forked and cloned separately. The mild exception to this concerns user and API document ...
```
Please change https://github.com/DlangScience/package to `https://github.com/DlangScience/<packageName>`, in verbatim/code (i.e. triple backticks)

> within the respective Github site

should be "within their respective Github sites"

> The mild exception to this concerns user and API documentation, as outlined in the documentation section below.

I don't understand. user and API documentation for each project is held within each independent repository.

GeraldJansen committed:
Update contrib_help.md

removed "The mild exception to this concerns user and API documentation, as outlined in the documentation section below." Still coming is a bit more documentation on the package-specific documentation - i.e. all projects need to structure their documentation in co-ordinated fashion (eg. site/*.md pages) so that all the documentation can be assembled coherently under a common web site (now DlangScience.github.io)

GeraldJansen committed:
Update contrib_help.md

John-Colvin commented:
Ok, I'm happy with this as it is. Let me know if you want to add more in this pull request, if not I'll merge it.

ghost commented:
Please merge.
